### ISSUE TITLE
New KS2 Lite shipping next month

### ISSUE THREAD

**Issue by 1Knightwing** [rdugan/iceriver-oc#89]
Got a couple ordered. Should I try your KS2 OC firmware on those or hold off until you've had a chance to scope the new unit out?
Sorry if this is an inappropriate place to ask but the only way I could think to do so.

**Comment by rdugan** [comment]
no, current fw definitely will not work.  I'll need time once they ship
